Pass new tag to version.yaml in terra-helmfile repo

smark88 opened the pull request.

Commit:
firt pass testing docker image

Commit:
fixed args

re try

testing reading datarepo version

test

added key path

added yq replacements

added inserts and read out

added commits now needs push

smark88 commented:
Just needs `git push` added

snf2ye commented on .github/workflows/alpha-promotion.yaml:
```diff
@@ -0,0 +1,49 @@
+        with:
+          args: yq w -i terra-helmfile/versions.yaml releases.datarepo.chartVersion ${{ env.chartVersion }}
+      - name: "Read terra-helmfile daterepo fields in versions.yaml"
+        uses: docker://mikefarah/yq:latest
```
Sorry to be lame, but is there a more descriptive docker path we could use?

smark88 replied:
i don't follow? mikefarah is the official releaser of yq
https://github.com/mikefarah/yq
https://hub.docker.com/r/mikefarah/yq

snf2ye replied:
Gotcha! I was wrong.

fboulnois reviewed (approved):
lgtm

Commit:
first try at making a pr

Commit:
persist-credentials: false

reworked bot and token

added branch before pr

indentation

syntax

need global configs

added debug

credentials test

trying new tag from forked repo

debug test path

tesst

removed hack stuff

idk

idk2

using new pat

updated

idk 3

test top level repo

changed path

end to end

smark88 commented:
This now creates automerge pr for tracking purposes vs pushing to master

https://github.com/broadinstitute/terra-helmfile/pull/92

@fboulnois @snf2ye please review

Commit:
changed to run nightly after tagger
